cbv3_django_prototype/cbv3_django_prototype/resources/models.py

```diff
@@ -1,3 +1,47 @@
+import uuid
+import django.contrib.postgres.fields as pg
 from django.db import models
+from django.utils import timezone
 
-# Create your models here.
+class Resource(models.Model):
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid1, editable=False)
```

Comment:
This field is implicitly defined by Django, so does not need to be part of our model definition.

Reply:
Disregard. I ser that we want to use UUID rather than sewuential IDs. ☺